wiki/Ranking_Criteria/osu!mania/de.md

```diff
@@ -1,91 +1,205 @@
+- **Bracket-Patterns sollten sparsam verwendet werden.**
+- **Lange Noten, die mit benachbarten, gleichzeitig gehaltenen Tasten gespielt werden, sollten gleichzeitig losgelassen werden.** Lange Noten mit mehreren Release-Zeiten erfordern eine Koordination, die Spieler dieses Niveaus nicht vernünftig bewältigen können.
 
-Mit USB verbundene Tastaturen können mehr als 6 Tasten gleichzeitig nicht erkennen. Diese Regel wurde erstellt um den Spielspaß aller sicherzustellen *Das Ende eines Hold's ist davon nicht betroffen denn sie werden losgelassen, nicht gedrückt*
+### ![](/wiki/shared/diff/hard-m.png?20211215) Hard
 
-- **Es ist nicht erlaubt zwei Noten auf der gleichen Column auf dem selben Timing zu platzieren. Es muss immer ein Abstand zur nächsten Note auf der gleichen Column bestehen.**
-- **Letterboxing muss ausgeschaltet sein.** Letterboxing wird in osu!mania nicht unterstützt und gehört deshalb ausgeschaltet.
+#### Richtlinien
 
-### Difficulty Spread
+- **HP und OD Werte sollten nicht höher als 8 sein.**
+- **Taktteilung von 1/8 und höher sollten nicht aufeinanderfolgend verwendet werden.** Diese können für Spieler dieses Niveaus körperlich zu anstrengend sein. Eine Ausnahme von dieser Leitlinie wäre die Verwendung von 3 Grace-Noten, die sich auf höherem Niveau befinden (1/8, 1/12 und 1/16), da sie nur zur Akzentuierung von Gracing-Klängen verwendet werden.
+- **Lange Streams von 1/4-Noten sollten mindestens eine 1/2-Pause enthalten, wo dies natürlich nach jeder musikalischen Phrase möglich ist.** Diese können für Spieler dieses Niveaus körperlich zu anspruchsvoll sein.
+- **Chords innerhalb eines 1/4-Streams sollten mindestens ein 1/1-Beat voneinander entfernt sein.** Eine dichtere Verwendung von Chords ist für Spieler dieses Niveaus zu schwierig.
+- **Vermeide die Verwendung von Chords während eines 1/6-Streams.** Jedoch ist die Verwendung von Chords am Anfang oder Ende eines Streams in Ordnung.
+- **Von der Verwendung von 1/4 Jacks wird abgeraten.** Diese können für Spieler dieses Niveaus körperlich zu anstrengend sein. Minijacks mit ausreichender Ruhezeit dazwischen können akzeptabel sein.
+- **Vermeide die Verwendung von mehr als 9 aufeinanderfolgenden Noten in einem Trill.** Die Koordination und das Gleichgewicht der Hände, die für dieses Pattern erforderlich sind, sind für Spieler auf diesem Niveau zu schwierig.
+- **Lange Noten sollten für mindestens 1/4 eines Beats gehalten werden.**
+- **Objekte, die direkt nach kurzen Slider-Velocity-Änderungen erscheinen, sollten mit der normalen Scroll-Geschwindigkeit der Beatmap erscheinen.** Um dies zu erreichen, sollten kurze Slider-Velocity-Änderungen nur in den leeren Bereichen zwischen den Noten verwendet werden und sich auf einen Abstand von 1,0x ausgleichen.
+- **Langzeitige Slider-Velocity-Änderungen sollten zwischen 0,8x und 1,10x liegen.**
+- **Auf Slider-Velocity-Gimmicks sollte man innerhalb von 1/2 Beat oder mehr reagieren können.**
```

Comment:
Just as a side note, I'm not sure about a good writing style for this word, so I think it is fine to leave as-is.